syft/tensor.py

```diff
@@ -1397,6 +1397,23 @@ def mm(self, tensor2):
 
         return syft.mm(self, tensor2)
 
+    def half(self):
+        """
+        casts the tensor to half-precision float type
+
+        ----------
```

Comment:
To adhere to the docstring format in #252, it should say 
```
Parameters
----------
```
even when there are no parameters